wasmbinding/bindings/msg.go

```diff
@@ -70,11 +70,11 @@ type AddAdmin struct {
 
 type AddAdminResponse struct{}
 
-type SubmitProposal struct {
-	Proposals Proposals `json:"proposals"`
+type SubmitAdminProposal struct {
+	AdminProposal AdminProposal `json:"admin_proposal"`
```

Comment:
why don't reuse `paramChange.ParameterChangeProposal` instead of defining our own `ParamChangeProposal`? just as we did with `paramChange.ParamChange` in the `ParamChangeProposal`

Reply:
because we may have more than one proposal in the future and they will all be relative to `AdminProposal`s